Userland/Libraries/LibJS/Runtime/GlobalObject.cpp

```diff
@@ -223,26 +223,41 @@ JS_DEFINE_NATIVE_FUNCTION(GlobalObject::is_finite)
 // 19.2.4 parseFloat ( string ), https://tc39.es/ecma262/#sec-parsefloat-string
 JS_DEFINE_NATIVE_FUNCTION(GlobalObject::parse_float)
 {
+    // NOTE: This is an optimization.
```

Comment:
Let's stick to the usual comment format:

```suggestion
    // OPTIMIZATION: If the argument is already a number, we can skip the roundtrip via ToString.
```

Reply:
We should go through and make this a real standard, because I copy-pasted this comment from elsewhere